context/app/static/js/pages/entity-search/DonorSearch/DonorSearch.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import SearchWrapper from 'js/components/entity-search/SearchWrapper';
+import { getDonorMetadataFields } from 'js/components/entity-search/SearchWrapper/utils';
+
+function DonorSearch() {
+  const entityType = 'donor';
```

Comment:
Maybe just inline it?

Reply:
It's beneficial as it's used in two places.